### PR TITLE
[+] CORE: Remove many smarty variables

### PR DESCRIPTION
**Pull request by julienbourdeau**
## Description

A lot of variable were still passed to Smarty, because we have a StarterTheme and many presenter, they shouldn't be used anymore.
## Steps to Test this Fix

There isn't step to reproduce but you can test _Classic_ and/or _StarterTheme_ and your moduleto make sure it doesn't break anything.
To be fair, it probably does since `objectSerializer` has been renamed.

### CODE REVIEW

**Comment by kpodemski** [comment]
@julienbourdeau are you sure that we don't need PS_CATALOG_MODE anymore? Is there any global variable for it?

**Comment by julienbourdeau** [comment]
@kpodemski No don't worry I'll put it back. I made the PR too early :sweat_smile:

File: classes/controller/FrontController.php [hunk 1]
```diff
@@ -857,6 +813,7 @@ public function setMedia()
         $this->addCSS([
             _THEME_CSS_DIR_ . 'theme.css',
             _THEME_CSS_DIR_ . 'custom.css',
+            _THEME_CSS_DIR_.'rtl.css',
```
Comment (mickaelandrieu) [inline classes/controller/FrontController.php]: rtl.css is for ... ?

Comment (julienbourdeau) [inline classes/controller/FrontController.php]: Oops ! Thanks

Comment (julienbourdeau) [inline classes/controller/FrontController.php]: I actually dont know how much style you need to make your website RTL.
There is a body class `lang-rtl` or `lang-ltr`, so you can override your styles, I guess the separate file only make sense if there is a lot to rewrite :confused:

Comment (mickaelandrieu) [inline classes/controller/FrontController.php]: ok, make sense :)

Comment (julienbourdeau) [inline classes/controller/FrontController.php]: I forgot to check if language is RTL :grimacing:

**Pull request merged**